driver/native.go

```diff
@@ -1,14 +1,74 @@
 package driver
 
-// AndroidContext is passed to the `RunNative` callback when it is executed on an Android device.
-// The VM, Env and Ctx pointers are reqiured to make various calls into JVM methods.
+// NativeWindow is an extension interface for `fyne.Window` that gives access
+// to platform-native features of application windows.
+//
+// Since: 2.5
+type NativeWindow interface {
+	// RunNative  provides a way to execute code within the platform-specific runtime context for a window.
+	// The context types are defined in the `driver` package and the specific context passed will differ by platform.
+	RunNative(func(context any) error) error
```

Comment:
All of the implementations for `RunNative()` just return the error from the inner function. Should the function really return an error?

Reply:
Got rid of the errors in the signature